### ISSUE TITLE
Enable "Delete' key while live recording. (shift+rec)

### ISSUE THREAD

**Issue by antf4rm** [polyend/TrackerIssues#1224]
Please enable the delete key during live recording.

Holding delete while in live record should clear pattern data while held.

This would add to the fluidity of live programming.

**Comment by trutypesounds** [comment]
Do "undo" works in rec arm mode?

**Comment by antf4rm** [comment]
> Do "undo" works in rec arm mode?
no, undo is not available in live record mode.

**Comment by UserOverflow** [comment]
Yes, must have! I was trying to create live song - not offline, but for jam. I can't stop, undo etc - I just need do everything when play pattern. This is *must have* for jam and real-time composing. Th same as "overwrite" which not work on a single pattern.
